packages/bot-web-ui/src/stores/quick-strategy-store.ts

```diff
@@ -100,6 +140,7 @@ export default class QuickStrategyStore implements IQuickStrategyStore {
         const selected_strategy = STRATEGIES[this.selected_strategy];
         const strategy_xml = await import(/* webpackChunkName: `[request]` */ `../xml/${selected_strategy.name}.xml`);
         const strategy_dom = Blockly.Xml.textToDom(strategy_xml.default);
+        this.addDinamicallyBlockToDOM('PREDICTION', 'last_digit_prediction', trade_type_cat, strategy_dom);
```

Comment:
Also we need to do it for onsubmit because we just compute data here onsubmit. But we don't reuse this computation again can we add this to reusable method ?

Reply:
`this.addDinamicallyBlockToDOM()` was created as an overused method so that we could use it in the future, now it is only used 1 time so far. What is the question or suggestion for `this.addDinamicallyBlockToDOM()` I don't get...

Reply:
what i meant was on click of `onsubmit` there were some methods that can be removed out the same way u did for `addDinamicallyBlockToDOM` to increase the readability just a suggestion.

Reply:
Please specify the subject of discussion, it is not clear which methods you would like to remove

Reply:
if you are talking about `modifyFieldDropdownValues`, `modifyValueInputs` then we can move them, but in my opinion, it has not a lot of sense because we aren't gonna use in any other places, only here..

Reply:
@maryia-matskevich-deriv yes i was talking about this entire `onSubmit` method so we can make it a bit smaller but lets not do as it may require more time in the sprint lets take it later 🙏 .